tensorflow/cc/experimental/libexport/load.h

```diff
@@ -83,16 +83,16 @@ class TFPackage {
   // Returns a BundleReader for reading variable values.
   //
   // This TFPackage retains ownership of the underlying reader.
-  tensorflow::BundleReader* GetVariableReader() {
-    return variable_reader_.get();
+  const std::unique_ptr<tensorflow::BundleReader> &GetVariableReader() {
```

Comment:
Thanks for the changes! Could you please revert this part back. In general we prefer to pass and return unique_ptr values, not pointers or references to unique_ptr.

Reply:
Thank you for reviewing my pull request and providing feedback. I apologize for the confusion, and I understand the preference for passing and returning unique_ptr values. I will revert this change as per your suggestion and resubmit my pull request. I appreciate your guidance and look forward to making this contribution align with the project's best practices.